.vscode/settings.json

```diff
@@ -2,5 +2,6 @@
 	"editor.insertSpaces": false,
 	"editor.detectIndentation": false,
 	"prettier.tabWidth": 4,
-	"prettier.useTabs": true
+	"prettier.useTabs": true,
+	"typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
what is the purpose of this configuration?

Reply:
This is injected when you manually configure the TypeScript version through the UI to use your local typescript version. This is part of the configuration steps for the TSLinter if I remember correctly.